### PR TITLE
Bone spear

### PR DESCRIPTION
**Pull request by jaenster**
A attack seq specifically for bone spear's

How to use? In your class config
```javascript
	// Load the bone custom class attack over the common Necromancer file
	Config.CustomClassAttack = 'Bone';
```

No need to setup AttackSkill / curses or anything else.

### CODE REVIEW

File: d2bs/kolbot/libs/GameData.js [hunk 1]
```diff
@@ -135,6 +135,7 @@ Object.freeze(AreaData);
...
...7", "2739"], ["466", "1480", "2825"], ["477", "1515", "2912"], ["489", "1549", "2998"], ["501", "1583", "3084"], ["513", "1617", "3170"], ["525", "1651", "3257"], ["539", "1685", "3343"], ["552", "1720", "3429"], ["565", "1753", "3515"], ["579", "1788", "3602"], ["592", "1821", "3688"], ["605", "1856", "3774"], ["618", "1891", "3860"], ["632", "1924", "3947"], ["645", "1959", "4033"], ["658", "1992", "4119"], ["673", "2027", "4205"], ["688", "2061", "4292"], ["702", "2095", "4378"], ["717", "2129", "4464"], ["732", "2163", "4550"], ["746", "2197", "4637"], ["761", "2232", "4723"], ["775", "2265", "4809"], ["790", "2300", "4895"], ["805", "2333", "4982"], ["821", "2368", "5068"], ["837", "2403", "5154"], ["853", "2436", "5240"], ["868", "2471", "5327"], ["884", "2504", "5413"], ["900", "2539", "5499"], ["916", "2573", "5585"], ["932", "2607", "5672"], ["948", "2641", "5758"], ["964", "2675", "5844"], ["982", "2709", "5930"], ["999", "2744", "6017"], ["1016", "2777", "6103"], ["1033", "2812", "6189"], ["1051", "2845", "6275"], ["1068", "2880", "6362"], ["1085", "2915", "6448"], ["1103", "2948", "6534"], ["1120", "2983", "6620"], ["1137", "3016", "6707"], ["10000", "10000", "10000"]],
```
Comment (noah-) [inline d2bs/kolbot/libs/GameData.js:138]: nit: hpLookup

File: d2bs/kolbot/libs/common/Attack.js [hunk 2]
```diff
@@ -276,6 +276,10 @@ var Attack = {
 				errorInfo = " (No valid attack skills)";
 
 				break;
+
```
Comment (noah-) [inline d2bs/kolbot/libs/common/Attack.js:279]: extra new line

File: d2bs/kolbot/libs/common/Attacks/Bone.js [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+		}
+		// 3) Need to dim someone's vision?
+		//ToDo actually write this properly
+
+		// 4) who is our merc attacking?
+		let merc = me.getMerc();
+		if (merc) {
+			const unit = getUnits(sdk.unittype.Monsters)
+				.filter(unit => {
+					if (!Attack.checkMonster(unit)) {
```
Comment (noah-) [inline d2bs/kolbot/libs/common/Attacks/Bone.js:50]: combine lines 50-62 with 
condition || condition || ..

would change fat arrow to regular function syntax as well

**Comment by theBGuy** [comment]
Are you going to come back to this? I like the general idea

**Pull request closed without merging**